### ISSUE TITLE
Require gradient false in eval mode

### ISSUE THREAD

**Issue by hwanheelee1993** [henryhungle/MTN#7]
In your code, you did not wrap the eval part with "torch.no_grad()", which means that your code computes gradient for validation data. That leads to unnecessary computation... You'd better edit that part.

**Comment by henryhungle** [comment]
Thanks! I updated the code and will close this
